ew/static/smelting.py

```diff
@@ -211,6 +211,35 @@
         },
         products=[ewcfg.weapon_id_pickaxe]
     ),
+    EwSmeltingRecipe(
```

Comment:
i don't see any code showing where this modifies mining yields, pretty funny if intentional but do you want to make the diamond pick useless?

Reply:
ew/cmd/juviecmd/juviecmds.py line 419 does it afaik

Reply:
pffff didn't see it my bad